server/.env

```diff
@@ -1 +1,2 @@
-TEAM_MEMBERS="Aecio, Bonnie, Shums Kassam, Simon"
+TEAM_MEMBERS="Aecio, Bonnie, Shums Kassam, Simon, Remi"
+MONGO_DB="mongodb://localhost/archibald"
```

Comment:
I now realize that I committed my .env file yesterday and that it got merged with dev in my frontend PR, yikes. :\

It's already .gitignored, maybe we should just remove it from the repo at this point.

Reply:
Absolutely, will do it on this branch